docs/sources/administration/image_rendering.md

```diff
@@ -194,3 +194,21 @@ it's available for the Grafana process, e.g.
 ```bash
 export GF_RENDERER_PLUGIN_CHROME_BIN="/usr/bin/chromium-browser"
 ```
+
+### Grafana image renderer plugin and certificate signed by internal CAs
```

Comment:
```suggestion
### Grafana image renderer plugin and certificate signed by internal CA's
```

Reply:
Hi @marefr, I just pushed a commit as you suggested. Generelly however I ask myself whether it really is a good idea to spell-check code blocks - If that policy has been in place all the time, how did the 'eror' in the log message pass it?

Regarding the second one - I'm not an authority about spelling of English abbreviations, but generally "CA's" would be genitive, while I interpret "CAs" as plural, which is what is meant. Probably the correct abbreviation should be CA ... so I pushed another one that avoids the issue entirely.